compiler/rustc_passes/src/liveness.rs

```diff
@@ -403,7 +403,7 @@ impl<'tcx> Visitor<'tcx> for IrMaps<'tcx> {
                 if let Some(captures) = self
                     .tcx
                     .typeck(closure_def_id)
-                    .closure_captures
+                    .closure_min_captures
                     .get(&closure_def_id.to_def_id())
                 {
                     // If closure captures is Some, upvars_mentioned must also be Some
```

Comment:
I have a feeling this here will cause issues when the feature is enabled. Because we will be possibly adding nodes for the same variable multiple times.

Reply:
Can you take some test portions from here: https://github.com/tmiasko/rust/blob/4dc56614b27337fe770943760f41a58bbab7c7c6/src/test/ui/liveness/liveness-upvars.rs

And convert them to use precise capture and add tests

Reply:
note for jenny: remove the move and it will work (first test in that file)

Reply:
This is stilll pending

Reply:
closure captures -> closure_min_captures

Reply:
@jenniferwills this comment still needs fixing